lib/java/com/google/android/material/card/MaterialCardViewHelper.java

```diff
@@ -165,6 +160,13 @@ void loadFromAttributes(@NonNull TypedArray attributes) {
     setCheckedIcon(
         MaterialResources.getDrawable(
             materialCardView.getContext(), attributes, R.styleable.MaterialCardView_checkedIcon));
+    Resources resources = materialCardView.getResources();
+    setCheckedIconSize(
```

Comment:
set it on the style instead

Reply:
Do you mean the default value?

Reply:
yeah sorry

yeah sorry that's what I meant

Reply:
fallback value should be 0

Reply:
Done